### ISSUE TITLE
Train.py not working 

### ISSUE THREAD

**Issue by HaeSung-Oh** [HustAlexander/MCWSS#2]
Thanks for your valuable work.

I have a question.

` loss2 =label2*(torch.min(cam_1,cam_2).mean(dim=1)-0.1*torch.max(cam_1,cam_2))`

The shape of the loss function does not match, so errors continue to occur.

To be calculated properly, I think it is correct to apply the mean function to torch.max as shown below.

` loss2 =label2*(torch.min(cam_1,cam_2).mean(dim=1)-0.1*torch.max(cam_1,cam_2).mean(dim=1))`

If I'm wrong, please correct me.

**Comment by cwb0611** [comment]
I also found this problem, I agree with your modification
